### DISCUSSION TITLE
Updating private npm packages (Nexus)

### DISCUSSION THREAD

**Discussion by enys** [renovatebot/renovate#14814] [Request Help] [answered]
I am having a hard time to get Renovate, hosted GitHub app to update scoped npm packages.
We use Nexus3 as private registry and npm proxy.

Working:

- Updating public npm packages via the nexus proxy ( third party and our own public packages (scoped))
- and thus authentication works

Not working:

- updating private scoped packages.

Log says:
```
DEBUG: Failed to look up dependency @linkurious/shared (@linkurious/shared)(packageFile="package.json", dependency="@linkurious/shared")
```
no specific error.

Renovate config:
```
{
  "extends": [
    "config:base"
  ],
  "baseBranches": ["develop-stable"],
  "labels": ["dependencies"],
  "prConcurrentLimit": 5,
  "respectLatest": false,
  "ignoreUnstable": false,
  "schedule": "every weekend",
  "vulnerabilityAlerts": {
    "labels": ["type:security"]
  },
  "npmrc": "registry=https://REDACTED/repository/npm/",
  "hostRules": [
    {
      "hostType": "npm",
      "matchHost": "https://REDACTED/repository/npm/",
      "encrypted": {
        "token": "REDACTED_TOKEN"
      },
      "authType": "Basic"
    }
  ]
}
```

Specificity, if that can have an impact, private packages don't have a 'latest' tag

Any ideas ?

**Comment by rarkins** [comment, marked as answer]
Indeed, Renovate was assuming the `latest` tag is always set. Fix: https://github.com/renovatebot/renovate/pull/14821

**Reply by enys** [reply]
Oh great, I agree it's a bit of an edge case, will test it as soon as it's out ! Thanks

**Reply by enys** [reply]
Thanks for this !

**Reply by rarkins** [reply]
Working now?

**Reply by enys** [reply]
I can confirm it works.
Thanks for the quick fix and release !
